cmd/snap/main.go

```diff
@@ -106,6 +107,27 @@ type parserSetter interface {
 	setParser(*flags.Parser)
 }
 
+func lintDesc(cmdName, optName, desc, origDesc string) {
+	if len(optName) == 0 {
+		logger.Panicf("option on %q has no name", cmdName)
+	}
+	if len(origDesc) != 0 {
+		logger.Panicf("description of %s's %q of %q set from tag (=> no i18n)", cmdName, optName, origDesc)
+	}
+	if len(desc) > 0 {
+		if !unicode.IsUpper(([]rune)(desc)[0]) {
```

Comment:
Do we need to check for "is this thing something that can be uppercase in the first place" instead?

Alternatively (pseudocode) `if upper(desc[0]) != desc[0] { ... }`